tox.ini

```diff
@@ -78,11 +78,32 @@ commands = sphinx-build -d "{toxworkdir}/docs_doctree" docs "{toxworkdir}/docs_o
            sphinx-build -d "{toxworkdir}/docs_doctree" docs "{toxworkdir}/docs_out" {posargs:-W --color -blinkcheck}
            python setup.py check -r -s
 
+
+[testenv:news]
+description = Invoke towncrier to update the NEWS file
+basepython = python3.7
+passenv = *
+deps = towncrier
+commands =
+    towncrier {posargs}
+
 [testenv:build]
-description = do a build
+description = Build and sign an sdist and bdist
 basepython = python3.7
 skip_install = true
 passenv = *
 deps = click >= 7.0
       pep517 >= 0.5.0
-commands = python release.py
+commands =
+    python release.py build
+    python release.py sign
```

Comment:
Probably we should make this conditional on `gpg` being installed with a default key set or something, because I just realized that this probably will (or at least *should*) fail on Travis.